### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: [https://edu-redmine.sun-asterisk.vn/issues/76168](https://edu-redmine.sun-asterisk.vn/issues/76168)

## WHAT (optional)
- Init project

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        environment:
+          MYSQL_DATABASE: db-test
+          MYSQL_USER: user-test
+          MYSQL_PASSWORD: password-test
+          MYSQL_ROOT_PASSWORD: password-test
+    before_script:
+      - bundle _2.1.4_ install --path vendor/bundle # _2.1.4_ if ruby 2.7.1 or 3.0.2
+    script:
+      - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
+      - bundle _2.1.4_ exec rspec # _2.1.4_ if ruby 2.7.1 or 3.0.2
```
Comment (huongnt-2545) [inline .sun-ci.yml]: comment script chạy rspec e nhé, P2 khi viết unit test sẽ bỏ comment sau nha

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RailsTutorial</title>
```
Comment (huongnt-2545) [inline app/views/layouts/application.html.erb]: I18n

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**